Unable to enable lz4 compression when installing with PECL

I started a new Docker container with `docker run --rm -it php:8.2-cli bash`, and I've tried to install redis 6.0.0RC1 with lz4 support, but I received a `only system liblz4 is supported` error:

```
> apt-get update
> apt-get install -qy liblz4-dev
> pecl install redis-6.0.0RC1

[...]
enable igbinary serializer support? [no] : no
enable lzf compression support? [no] : no
enable zstd compression support? [no] : no
enable msgpack serializer support? [no] : no
enable lz4 compression? [no] : yes
[...]
checking whether to enable lz4 compression... yes
checking use system liblz4... no
checking for hash includes... /usr/local/include/php
checking for json includes... /usr/local/include/php
checking for redis json support... enabled
checking for redis igbinary support... disabled
checking for pkg-config... /usr/bin/pkg-config
configure: error: only system liblz4 is supported
ERROR: `/tmp/pear/temp/redis/configure --with-php-config=/usr/local/bin/php-config --enable-redis-igbinary=no
   --enable-redis-lzf=no --enable-redis-zstd=no --enable-redis-msgpack=no --enable-redis-lz4=yes' failed
```

--- a/package.xml
+++ b/package.xml
@@ -271,6 +271,7 @@ http://pear.php.net/dtd/package-2.0.xsd">
   <configureoption name="enable-redis-zstd" prompt="enable zstd compression support?" default="no"/>
   <configureoption name="enable-redis-msgpack" prompt="enable msgpack serializer support?" default="no"/>
   <configureoption name="enable-redis-lz4" prompt="enable lz4 compression?" default="no"/>
+  <configureoption name="with-liblz4" prompt="use system liblz4?" default="yes"/>
  </extsrcrelease>
  <changelog>
  <release>

